CompetitionTemplate/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -147,6 +146,7 @@ public void setupHangerCommands(
         oi.operatorButtons.getifAvailable(12).whileHeld(winchRetract);
         
         winchFollow.includeOnSmartDashboard();
+        scale.includeOnSmartDashboard();
```

Comment:
We probably want an operator button for this? Maybe while-held so we can stop if we need to panic?